### ISSUE TITLE
Used old version when installed with composer require

### ISSUE THREAD

**Issue by shawe** [mybizna/base#1]
I'm following your guide from https://mybizna.gitbook.io/mybizna-erp/readme/how-to-install-in-laravel/how-to-install-as-new-laravel-project

When you do a clean new installation of mybizna ```composer create-project mybizna/setup mybizna``` and after that you do something like this:

```
composer require mybizna/account
composer require mybizna/assets
composer require mybizna/automigrator
composer require mybizna/base
composer require mybizna/bill
composer require mybizna/calendar
composer require mybizna/core
composer require mybizna/discussion
composer require mybizna/expense
composer require mybizna/fleet
composer require mybizna/hrm
composer require mybizna/mail
composer require mybizna/mrp
composer require mybizna/partner
composer require mybizna/sale
```

All modules packages are getting version 23.11 or greater (but not latest version of each module, I check it manually), but base module is blocked in version 1.2.19 (released at Jan 6, 2023) meanwhile latest version are 24.03.009.

This was causing that some newer files are not founded and code results broken.

It seems that latests changes are not tested from a clean installation and this have some issues. Like this https://github.com/mybizna/product/pull/1 or this:

![image](https://github.com/mybizna/base/assets/145989/3efb1a73-f708-4549-9408-1484047bc72d)

If I try to force that mybizna/base use ^24.01 the requirements could not be resolved, some packages require newest version, but if you try to increase it, the problems are also increasing without easy solution.

**Comment by dedanirungu** [comment]
You can try Step-by-Step Setup as I work to fix the issue.

https://mybizna.gitbook.io/mybizna-erp/readme/how-to-install-in-laravel/how-to-install-as-new-laravel-project

**Comment by shawe** [comment]
> You can try Step-by-Step Setup as I work to fix the issue.
> 
> https://mybizna.gitbook.io/mybizna-erp/readme/how-to-install-in-laravel/how-to-install-as-new-laravel-project

No, doesn't work, I yet tested before create the issue. mybizna/base always download and old version.

**Comment by dedanirungu** [comment]
On my side is loading the latest version when I set it up either way. I have updated https://mybizna.gitbook.io/mybizna-erp/readme/how-to-install-in-laravel/how-to-install-as-new-laravel-project to laravel 10. Try to set it and let me know your result.
 
```
  - Installing composer/installers (v1.12.0): Extracting archive
  - Installing psr/cache (3.0.0): Extracting archive
  - Installing doctrine/event-manager (2.0.0): Extracting archive
  - Installing doctrine/deprecations (1.1.3): Extracting archive
  - Installing doctrine/cache (2.2.0): Extracting archive
  - Installing doctrine/dbal (3.8.3): Extracting archive
  - Installing wildside/userstamps (2.4.0): Extracting archive
  - Installing spatie/laravel-permission (6.4.0): Extracting archive
  - Installing nwidart/laravel-modules (10.0.6): Extracting archive
  - Installing mybizna/automigrator (24.02.001): Extracting archive
  - Installing mybizna/assets (24.03.014): Extracting archive
  - Installing laravel/ui (v4.5.0): Extracting archive
  - Installing mybizna/base (24.03.009): Extracting archive
  - Installing mybizna/core (24.03.002): Extracting archive
  - Installing mybizna/partner (24.02.006): Extracting archive
  - Installing mybizna/account (24.03.001): Extracting archive

```

**Comment by shawe** [comment]
> On my side is loading the latest version when I set it up either way. I have updated https://mybizna.gitbook.io/mybizna-erp/readme/how-to-install-in-laravel/how-to-install-as-new-laravel-project to laravel 10. Try to set it and let me know your result.
> 
> ```
>   - Installing composer/installers (v1.12.0): Extracting archive
>   - Installing psr/cache (3.0.0): Extracting archive
>   - Installing doctrine/event-manager (2.0.0): Extracting archive
>   - Installing doctrine/deprecations (1.1.3): Extracting archive
>   - Installing doctrine/cache (2.2.0): Extracting archive
>   - Installing doctrine/dbal (3.8.3): Extracting archive
>   - Installing wildside/userstamps (2.4.0): Extracting archive
>   - Installing spatie/laravel-permission (6.4.0): Extracting archive
>   - Installing nwidart/laravel-modules (10.0.6): Extracting archive
>   - Installing mybizna/automigrator (24.02.001): Extracting archive
>   - Installing mybizna/assets (24.03.014): Extracting archive
>   - Installing laravel/ui (v4.5.0): Extracting archive
>   - Installing mybizna/base (24.03.009): Extracting archive
>   - Installing mybizna/core (24.03.002): Extracting archive
>   - Installing mybizna/partner (24.02.006): Extracting archive
>   - Installing mybizna/account (24.03.001): Extracting archive
> ```

I will try it again, yesterday when I send you a little fix doesn't work in any way I try it.

**Comment by shawe** [comment]
Seems to work fine now, but it's the same that I tested some times before this report.

The missing step before step 13 is run this commands:
```
npm install
npm run build
```

**Comment by dedanirungu** [comment]
There is no need for ```npm run build``` because all necessary assets are downloaded and saved under public/mybizna. This was needed to make installation easy.
